jane/doc/extensions/_01-stack-allocation/reference.md

```diff
@@ -60,7 +60,7 @@ including:
 ### Runtime behavior
 
 At runtime, stack allocations do not take place on the function call stack, but on a
-separately-allocated stack that follows the same layout as the OCaml
+separately-allocated stack that follows the same layout as the stock OCaml
```

Comment:
Sorry to be picky here, but I don't think we should add "stock" to all uses of OCaml. Putting the word in here suggests (to me) that there is a difference between the minor heap layout in OCaml vs OxCaml. We could be pedantic and write `O(x)Caml` here, but I'd prefer not so. Maybe "as the OCaml minor heap (unchanged in OxCaml)"?